gtda/images/filtrations.py

```diff
@@ -279,7 +280,7 @@ def fit(self, X, y=None):
         self : object
 
         """
-        X = check_array(X,  ensure_2d=False, allow_nd=True)
+        X = check_array(X, ensure_2d=False, allow_nd=True)
```

Comment:
Isn't it strange that `flake8` didn't fail on these double spaces before?

Reply:
Yes, it is!

Reply:
I'll investigate